odoo-shippable/Dockerfile

```diff
@@ -60,7 +60,8 @@ RUN apt-get remove -y git \
         poppler-utils antiword p7zip-full expect-dev mosh bpython \
         ghostscript graphviz openssh-server zsh \
         lua50 liblua50-dev liblualib50-dev \
-        exuberant-ctags \
+        exuberant-ctags python-dev gcc g++ libtiff5-dev libjpeg8-dev zlib1g-dev \
```

Comment:
Because in the odoo80 image I put in a different Variable the build dependencies of python packages so that we can install them and then uninstall at the end once the packages have been compiled. So, at that moment in the odoo-shippable image, those packages are not installed.